src/test/java/org/assertj/core/internal/strings/Strings_assertContains_Test.java

```diff
@@ -63,6 +64,17 @@ void should_pass_if_actual_contains_sequence() {
     strings.assertContains(someInfo(), "Yoda", "Yo");
   }
 
+  @Test
+  void should_fail_if_actual_does_not_contain_any_sequence() {
+    assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> strings.assertContainsAnyOf(someInfo(), "Yoda", "Luke", "Leia"))
+                                                   .withMessage(shouldContainAnyOf("Yoda", array("Luke", "Leia"), comparisonStrategy).create());
+  }
+
+  @Test
+  void should_pass_if_actual_contains_any_sequence() {
+    strings.assertContainsAnyOf(someInfo(), "Master Yoda", "Yoda", "Luke");
+  }
+
```

Comment:
Being a new assertion method, it would be better to have the tests in a separate `Strings_assertContainsAnyOf_Test` class.